### ISSUE TITLE
Link in description is problematic

### ISSUE THREAD

**Issue by TEUNGKU-ZULKIFLI** [maxrpark/R3F_smart_watch#1]
### **link not working. You can change from "serpia" to "r3f".**
**thank you see you again** `🤫🧏‍♂️🗿`

**Comment by maxrpark** [comment]
Hello, I am sorry I am not sure what link is not working.
Would you mind to explain with more details?

**Comment by TEUNGKU-ZULKIFLI** [comment]
Of course. The accessible link is https://smart-watch-r3f.vercel.app/, but in the repository 'About' section, the link is https://smart-watch-sepia.vercel.app/.
